src/cleo/_utils.py

```diff
@@ -70,9 +69,7 @@ def find_similar_names(name: str, names: list[str]) -> list[str]:
 
     # Only keep results with a distance below the threshold
     distance_by_name = {
-        key: value
-        for key, value in distance_by_name.items()
-        if value[0] < 2 * threshold
+        key: value for key, value in distance_by_name.items() if value[0] > threshold
     }
     # Display results with shortest distance first
     return sorted(distance_by_name, key=lambda key: distance_by_name[key])
```

Comment:
Side thought, this entire function could use `bisect.insort` to immediately build a list of similar names without this at the end, without that repeated `O(n)` hassle.